src/ray/util/filesystem.cc

```diff
@@ -85,4 +91,20 @@ std::string GetUserTempDir() {
   return result;
 }
 
+std::string GetStderrFile() {
+#ifdef __linux__
+  struct stat st;
+  if (!fstat(2, &st) && S_ISREG(st.st_mode)) {
+    char filepath[PATH_MAX + 1];
+    ssize_t len = ::readlink("/proc/self/fd/2", filepath, sizeof(filepath) - 1);
```

Comment:
Can you add a comment to indicate here we read the stderr file by the fd#2 ?

Reply:
I have added comment